internal/librarian/init.go

```diff
@@ -132,3 +139,72 @@ func setReleaseTrigger(state *config.LibrarianState, libraryID, libraryVersion s
 		library.ReleaseTriggered = trigger
 	}
 }
+
+// getLibraryChanges gets the changelog for the library with a non-empty libraryID,
+// if provided; or for all libraries if the libraryID is empty since the library
+// was last released.
+func getLibraryChanges(repo gitrepo.Repository, state *config.LibrarianState, libraryID string) error {
+	for i, library := range state.Libraries {
```

Comment:
I feel that this flow control for all libraries vs. one can happen a bit upstream, perhaps in `runInitCommand`. So `setReleaseTrigger` and `getLibraryChanges` don't need to repeat it?